docs/pipelines-in-pipelines.md

```diff
@@ -0,0 +1,106 @@
+<!--
+---
+linkTitle: "Pipelines in Pipelines"
+weight: 406
+---
+-->
+
+# Pipelines in Pipelines
+> 🌱 Pipelines in Pipelines is an alpha feature. The enable-api-fields feature flag must be set to "alpha" to specify PipelineRef or PipelineSpec in a PipelineTask. This feature is in Preview Only mode and not yet supported/implemented
+
+* [Proposal](https://github.com/tektoncd/community/blob/main/teps/0056-pipelines-in-pipelines.md "Proposal")
+* [Overview](#overview)
+* [Specification](#specification)
+* [Parameters](#parameters)
```

Comment:
Please update the TOC with the latest updated titles.

Also, we can add one line explanation to refer to the proposal in instead of adding it as link in a TOC. Is it clear?  Something similar to 

```
Pipeline in pipelines support in `pipelineTask` was designed and discussed in TEP-0056.
```

This statement can go under `Overview` or in the end of the doc.

Reply:
An example:

https://github.com/tektoncd/pipeline/blob/main/docs/pipelines.md#specifying-a-custom-task-spec-in-line-or-embedded

Reply:
Done